js/bter.js

```diff
@@ -63,7 +63,12 @@ module.exports = class bter extends Exchange {
 
     async fetchMarkets () {
         let response = await this.publicGetMarketinfo ();
-        let markets = response['pairs'];
+        let markets = undefined;
+        if ('pairs' in response) {
+            markets = response['pairs'];
+        } else {
+            markets = [];
```

Comment:
I guess, I'm going to add / throw an exception here, what do you think?

Reply:
Makes sense